kong-prometheus-plugin-0.4.0-1.rockspec

```diff
@@ -1,9 +1,9 @@
 package = "kong-prometheus-plugin"
-version = "0.3.4-2"
+version = "0.4.0-1"
```

Comment:
I think this should be `0.4.0-2`.
`-2` indicates the revision of the luarocks file itself. We had some trouble when we initially released this plugin and `-2` has stuck around since then.

Reply:
@hbagdi it is two separate commits

Reply:
Yes, `kong/kong` version needs to be bumped, but first I go through all these external plugins and the PR `kong/kong`.

Reply:
My bad regarding the separate commits. 
Could you update this to `0.4.0-2`?

Reply:
This is a totally new version `0.4.0-1` from `0.3.x` series, so it will not need `-2` as this is just a new release, as there is no `0.4.0` releases before this.